tools/MUXPGODatabase/restore-pgodb.ps1

```diff
@@ -0,0 +1,45 @@
+. .\version.ps1
```

Comment:
For lab builds, this script is explicitly invoked from the yml.

How does this work for local builds? We should not require devs to have to manually invoke a script. We should make it so that the right things happen as part of the VS build.

Reply:
Do we care to optimize local builds for everyone?  It will only happen on release bits, it will add to compile time and potentially make debugging more difficult.  It seems that in vast majority of cases it wouldn't provide much value.  If this is a one-off scenario then maybe running a command is ok?

Reply:
It is not a hard requirement. It is better if local builds produce the same results as lab builds. But we also don't want to force everyone to always run a script before building. If we have to choose, I would say making it so that local Release builds are not PGO-optimized is the preference.